src/main/java/com/deavensoft/paketomat/dispatcher/DispatcherModel.java

```diff
@@ -4,8 +4,6 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Data;
 import lombok.NoArgsConstructor;
-
-
 import javax.persistence.*;
 
 @NoArgsConstructor
```

Comment:
`   public DispatcherModel(@JsonProperty("id") Long id,@JsonProperty("email") String email, @JsonProperty("name") String name){
          this.id = id;
          this.email = email;
          this.name = name;
     }`

- [ ] Delete it